dev/com.ibm.websphere.appserver.features/visibility/public/jaspic-1.1/com.ibm.websphere.appserver.jaspic-1.1.feature

```diff
@@ -13,7 +13,8 @@ Subsystem-Name: Java Authentication SPI for Containers 1.1
 -features=\
   com.ibm.websphere.appserver.appSecurity-2.0; ibm.tolerates:=3.0, \
   com.ibm.websphere.appserver.internal.optional.jaxb-2.2; ibm.tolerates:=2.3, \
-  com.ibm.websphere.appserver.servlet-3.0; ibm.tolerates:="3.1, 4.0"
+  com.ibm.websphere.appserver.servlet-3.0; ibm.tolerates:="3.1, 4.0", \
+  com.ibm.websphere.appserver.eeCompatible-7.0; ibm.tolerates:="6.0,8.0"
```

Comment:
I am wondering if this should be 6.0 and tolerate 7.0 and 8.0 since it is servlet 3.0 (EE 6), tolerates 3.1 (EE 7) and 4.0 (EE8).  I don't think you are supposed to mix the order with different features because it can cause issues.

Reply:
Yes, it should be 6.0, tolerating 7 and 8. I will change this.

Reply:
You are correct. I will change this.